### ISSUE TITLE
snake doesnt stop at end of the map

### ISSUE THREAD

**Issue by chris-hawley** [iiegor/slither#26]
1 of 2 problems could be occuring

1. the map isnt loading 
2. the snake isnt stopping

![image](https://cloud.githubusercontent.com/assets/9792742/14727759/212c625a-07fc-11e6-89e1-077c5ef2d503.png)

**Comment by iiegor** [comment]
Yes, this is a known issue. I will try to fix this soon as posible.
